laravel-app/app/Models/User.php

```diff
@@ -2,13 +2,14 @@
 
 namespace App\Models;
 
-// use Illuminate\Contracts\Auth\MustVerifyEmail;
```

Comment:
also this one

Reply:
By default it is commented, I removed it at first  because it was unnecessary, but now it is needed so I brought it back sir.